tables/audisp/src/processeventstableplugin.cpp

```diff
@@ -185,15 +178,16 @@ Status ProcessEventsTablePlugin::generateRow(
   auto time_value = static_cast<std::int64_t>(current_timestamp.count());
 
   row["time"] = time_value;
-  row["syscall"] = syscall_name;
-  row["pid"] = syscall_data.process_id;
-  row["parent"] = syscall_data.parent_process_id;
+  row["syscall"] = syscall;
```

Comment:
The previous version would have only allocated memory for the `string` here, so we should at least `std::move(syscall)` to not needlessly create copies.

Reply:
nitpick: 'syscall' is usually an unfortunate name since it clashes with the Linux syscall() function (GitHub's syntax highlighting is showing it as a special symbol too)